Create vars.md (#2450)
Codecademy/docs: content/python/concepts/built-in-functions/terms/vars/vars.md

---
Title: 'vars()'
Description: 'Returns the __dict__ attirbute of an object.'
Subjects: 
	- 'computer science'
Tags: 
	- 'built-in functions'
CatalogContent: 
	- 'learn-python'
	- 'paths/computer-science'
---

**vars()** is a built-in python function that returns the __dict__ attribute of an object. The __dict__ attribute is a dictionary containing the object's changeable attributes.

## Syntax
```pseudo
vars(object)
```

## Example

> Use vars() to return the __dict__ attributes of the object 'Person'

Reviewer: ```suggestion
The following example uses `vars()` to return the `__dict__` attributes of the object `Person`:
```

```codebyte/python
	class Person:
		name = "John"
		age = 36
		country = "norway"

	x = vars(Person)
	

> 	>>>{'__module__': '__main__', 'name': 'John', 'age': 36, 'country': 'norway', '__dict__': <attribute '__dict__' of 'Person' objects>, '__weakref__': <attribute '__weakref__' of 'Person' objects>, '__doc__': None}

Reviewer: ```suggestion
```
That is not actual code or part of the codebyte. Remove it. Also add a line to output the value of `x` to the user.

---
Title: 'vars()'

> Description: 'Returns the __dict__ attirbute of an object.'

Reviewer 2: ```suggestion
Description: 'Returns the __dict__ attribute of an object.'
```

Subjects: 
  - 'Computer Science'
Tags: 
  - 'Functions'
  - 'Attributes'
  - 'Objects'
CatalogContent: 

>   - 'learn-python'

Reviewer 2: ```suggestion
  - 'learn-python-3'
```

  - 'paths/computer-science'
---

> **`vars()`** is a built-in python function that returns the `__dict__` attribute of an object. The `__dict__` attribute is a dictionary containing the object's changeable attributes.

Reviewer 2: ```suggestion
**`vars()`** is a built-in Python function that returns the `__dict__` attribute of an object. The `__dict__` attribute is a dictionary containing the object's mutable attributes.
```

## Syntax

```pseudo
vars(object)
```

> Here, 'object' refers to an instance of a particular class.

Reviewer 2: ```suggestion
Here, `object` refers to an instance of a particular class.
```

> ## Example

Reviewer: ```suggestion
## Codebyte Example
```

The following example uses `vars()` to return the `__dict__` attributes of the object `Person`:

```codebyte/python

> 	class Person:

Reviewer 2: the code should be shifted to the left
